acceptance/acceptance_test.go

```diff
@@ -484,6 +485,32 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	}
 }
 
+func hasKey(list []string, key string) bool {
```

Comment:
nit: `list` can be renamed to something more descriptive to reflect what's in that list. for consistency, it can be named `env` as in the method below

Reply:
I agree 'env' is slightly better since it's not any list that is accepted, a028dcf4a16881b2101ad65ce9d53b41938c6eb5